go.mod

```diff
@@ -238,7 +238,7 @@ replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible
 	github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.40.0
 	github.com/mtrmac/gpgme => github.com/mtrmac/gpgme v0.1.1
-	github.com/openshift/machine-config-operator => github.com/openshift/machine-config-operator v0.0.1-0.20230807154212-886c5c3fc7a9 // release-4.14
+	github.com/openshift/machine-config-operator => github.com/openshift/machine-config-operator v0.0.1-0.20240404192517-42218f968a9e // release-4.14
```

Comment:
I am having trouble bumping to latest version of MCO so for now updated it to a more upto date version of release-4.14 while I am figuring out bump to 4.16.

Reply:
Yeah I wonder if we need to first bump openshift/api openshift/client-go first removing MCO as a dep entirely.

Reply:
Are you suggesting that I remove commit that updates MCO deps from this PR?
Removing MCO dependency entirely doesn't seem right to me...

Reply:
From what I can tell the MCO dependency in this repo will be replaced by openshift/api and openshift/client-go because your only dependant on those types which the new MCO doesn't export anymore. So my question is does that update ned to happen before k8s. I am going to test it in another PR on top of this now.

Reply:
https://github.com/openshift/cluster-node-tuning-operator/pull/1021 this is built on your PR

Reply:
Ah, I understand what you meant now :)

Reply:
I got to the root of the failure this file is manually maintained and I think we just need to update the new API field

https://github.com/openshift/cluster-node-tuning-operator/pull/1021/commits/23ff3edb5be20c72a417ab1cefec2c872ac81b70

Reply:
passes now just needed to update everywhere https://github.com/openshift/cluster-node-tuning-operator/pull/1021

Reply:
Thanks for the deep dive! Updating my now.